server/data/collections/dataset_collection_local.yaml

```diff
@@ -15,14 +15,14 @@ datasets:
   database_type: "S3_DB"
   s3_bucket: "example"
   s3_key: "data/titanic.csv"
-  endpoint_url: "http://minio:9000"
+  endpoint_url: "http://minio:10000"
```

Comment:
I am curious, why did it need to be modified ?

Reply:
we had conflicts with port 9000 (usually with ipykernel when using jupyter notebook in vscode)